lib/spack/spack/binary_distribution.py

```diff
@@ -524,7 +524,7 @@ def extract_tarball(spec, filename, allow_root=False, unsigned=False,
     # Delay creating spec.prefix until verification is complete
     # and any relocation has been done.
     else:
-        install_tree(workdir, spec.prefix, symlinks=True)
+        shutil.copytree(workdir, spec.prefix, symlinks=True)
```

Comment:
why do you move AWAY from install_tree? Isn't the idea to use only install_tree?

Reply:
This was due to a bug in `distutils.copy_tree`. Now that we've switched to manual installations, I can switch this back.